cypress/support/reusable_tests/admin.js

```diff
@@ -87,31 +88,40 @@ Cypress.Commands.add('testAdminPageElements',(viewport,creds)=>{
 Cypress.Commands.add('testAdminFilterUsers',(viewport,creds)=>{
     cy.viewport(viewport);
     cy.login(creds.email,creds.password);
-
+    var type = 'lawyer';
```

Comment:
any particular reason we're using the lawyer user type here?

Reply:
Just one of the options available that is not a regular user.

Reply:
Do you want me to test with a random selection of the options? I can do that and not test with a static selection.

Reply:
I added all possible types and tested it